pulsar-broker/src/test/java/org/apache/pulsar/broker/service/BrokerServiceTest.java

```diff
@@ -172,8 +172,6 @@ public void testShutDownWithMaxConcurrentUnload() throws Exception {
             assertTrue(e instanceof PulsarClientException.TimeoutException);
         }
 
-        pulsar = null;
```

Comment:
I agree that this can be removed, but I am not seeing the connection between this line of code and the `SimpleProducerConsumerTestStreamingDispatcherTest` test. Can you add more detail?

Reply:
`BrokerServiceTest` and `SimpleProducerConsumerTestStreamingDispatcherTest` both are subclass of `MockedPulsarServiceBaseTest`.  So the conection is that they will call the same `MockedPulsarServiceBaseTest#pulsar`.

In addition, `MockedPulsarServiceBaseTest` has other subclasses also call `MockedPulsarServiceBaseTest#pulsar`, so here we'd better remove this line. @michaeljmarshall

Reply:
Thanks for your explanation. Do we run these tests in parallel? It seems like we shouldn't if they are sharing state.